test/addons/rbd-mirror/start

```diff
@@ -99,21 +118,29 @@ def configure_rbd_mirroring(cluster, peer_info):
     patch = {"spec": {"mirroring": {"peers": {"secretNames": [peer_info["name"]]}}}}
     kubectl.patch(
         "cephblockpool",
-        POOL_NAME,
+        POOL_NAMES[0],
         "--type=merge",
         f"--patch={json.dumps(patch)}",
         "--namespace=rook-ceph",
         context=cluster,
     )
 
-    print("Creating VolumeReplicationClass")
-    template = drenv.template("start-data/vrc-sample.yaml")
-    yaml = template.substitute(cluster=cluster, scname="rook-ceph-block")
-    kubectl.apply("--filename=-", input=yaml, context=cluster)
-
-    template = drenv.template("start-data/vgrc-sample.yaml")
-    yaml = template.substitute(cluster=cluster, pool=POOL_NAME, scname="rook-ceph-block")
-    kubectl.apply("--filename=-", input=yaml, context=cluster)
+    for storage_class in STORAGE_CLASSES_MAP:
+        print("Creating VolumeReplicationClass")
+        template = drenv.template("start-data/vrc-sample.yaml")
+        yaml = template.substitute(
+            cluster=cluster, vrcname=storage_class["vrc"], scname=storage_class["name"]
+        )
+        kubectl.apply("--filename=-", input=yaml, context=cluster)
+
+        template = drenv.template("start-data/vgrc-sample.yaml")
+        yaml = template.substitute(
+            cluster=cluster,
+            pool=storage_class["pool"],
+            vgrcname=storage_class["vgrc"],
+            scname=storage_class["name"],
+        )
+        kubectl.apply("--filename=-", input=yaml, context=cluster)
```

Comment:
I don't want to complicate the code like this. Lets keep the code simple, creating and setting up mirroring on for single pool.

To create multiple vrc/vgrc/sc and setup mirroring on all pools, we can run the entire code in parallel with 2 configurations. Each thread will create resources and set up mirroring for single pool.